### PR TITLE
added sqlite support

### PR DESCRIPTION
**Pull request by deepchoudhery**
- added sqlite support for ef scaffolders
- added global tool helper for windows development
- added some tests

### CODE REVIEW

File: src/VS.Web.CG.Mvc/Common/EFValidationUtil.cs [hunk 1]
```diff
@@ -10,13 +10,14 @@ namespace Microsoft.VisualStudio.Web.CodeGenerators.Mvc
 {
     internal static class EFValidationUtil
     {
+        const string EfDesignPackageName = "Microsoft.EntityFrameworkCore.Design";
+        const string SqlServerPackageName = "Microsoft.EntityFrameworkCore.SqlServer";
+        const string SqlitePackageName = "Microsoft.EntityFrameworkCore.Sqlite";
```
Comment (bricelam) [inline src/VS.Web.CG.Mvc/Common/EFValidationUtil.cs]: Might be better to check for Microsoft.EntityFrameworkCore.Sqlite.**Core** (but still recommend installing Microsoft.EntityFrameworkCore.Sqlite)

Comment (bricelam) [reply src/VS.Web.CG.Mvc/Common/EFValidationUtil.cs]: The main package depends on .Core and another package to pull in the native SQLite binaries:

* Project
  * Microsoft.Data.Sqlite
    * Microsoft.Data.Sqlite.Core
    * SQLitePCLRaw.bundle_e_sqlite

But you can use other SQLite binaries (like SQLCipher for encryption) by depending directly on .Core and a different bundle:

* Project
  * Microsoft.Data.Sqlite.Core
  * SQLitePCLRaw.bundle_e_sqlcipher

Comment (deepchoudhery) [reply src/VS.Web.CG.Mvc/Common/EFValidationUtil.cs]: gotcha. so really Sqlite.Core and SQlitePCLRaw.bundle_e_sqlite should be installed as well when we wanna use sqlite

Comment (deepchoudhery) [reply src/VS.Web.CG.Mvc/Common/EFValidationUtil.cs]: is checking for SQLitePCLRaw.bundle_e_sqlite or SQLitePCLRaw.bundle_e_sqlcipher necessary or do we leave that up to the user at that point?

File: src/VS.Web.CG.Mvc/Common/EFValidationUtil.cs [hunk 2]
```diff
@@ -26,5 +29,27 @@ internal static void ValidateEFDependencies(IEnumerable<DependencyDescription> d
                     string.Format(MessageStrings.InstallEfPackages, $"{EfDesignPackageName}, {SqlServerPackageName}"));
             }
         }
+
+        internal static void ValidateSQLiteDependency(IEnumerable<DependencyDescription> dependencies)
+        { 
+            var isSqlitePackagePresent = dependencies
```
Comment (vijayrkn) [inline src/VS.Web.CG.Mvc/Common/EFValidationUtil.cs]: Update per the message from Brice

File: src/VS.Web.CG.Mvc/Common/EFValidationUtil.cs (hunk 1 above)
Comment (bricelam) [reply src/VS.Web.CG.Mvc/Common/EFValidationUtil.cs]: Leave it up to the user. The code will compile just fine without it

**Pull request merged**